package.json

```diff
@@ -8,7 +8,7 @@
   "description": "steemit.com is the koa web server & middleware and react.js in-browser code for the world's first blockchain content + social media monetization platform!",
   "main": "index.js",
   "scripts": {
-    "build": "NODE_ENV=production ./node_modules/babel-cli/bin/babel-node.js ./node_modules/.bin/webpack --config ./webpack/prod.config.js; rm -rf ./lib; babel src --out-dir lib -Dq",
+    "build": "NODE_ENV=production ./node_modules/babel-cli/bin/babel-node.js ./node_modules/.bin/webpack --config ./webpack/prod.config.js; rm -rf ./lib; NODE_ENV=production babel --plugins transform-runtime,transform-inline-environment-variables src --out-dir lib -Dq",
```

Comment:
Not your code I know, but some questions:

- this could just be `babel-node` without the long path. and why do we need to invoke the webpack binary via `babel-node`? using `babel-node` in production is an antipattern.

Reply:
webpack config uses some fancy est6 syntax, probably we can refactor it to use plain js

Reply:
What babel-node without long path would it use - global babel-node or the one that is in node_modules?

Reply:
```
➜ dev mkdir foo
➜ dev cd foo
➜  foo npm install -g babel-cli
[snip]
+ babel-cli@6.26.0
added 233 packages in 49.679s
➜  foo which babel-node
/Users/goldibex/Library/Homebrew/bin/babel-node
➜  foo echo 'echo "BOOM" && exit 1' > /Users/goldibex/Library/Homebrew/bin/babel-node
➜  foo babel-node
BOOM
➜  foo npm init
[snip]
➜  foo npm install --save babel-cli
[snip]
➜  foo cat package.json | jq '.scripts.boom = "babel-node"' | tee package.json
[snip]
➜  foo npm run boom

> foo@1.0.0 boom /Users/goldibex/Development/steemit/foo
> babel-node

> 
```

Reply:
Node >= 8 supports object splats, so the only problem is the import/export syntax. 

Just so you know I'm not blowing smoke, check the output of `time` when babel-node is involved vs. when it isn't:

```
yarn build  47.55s user 1.89s system 112% cpu 43.801 total
yarn build  45.33s user 1.51s system 116% cpu 40.179 total
```

Shaves three seconds off the production build, well worth the effort.

Reply:
It doesn't use babel-node in production, `build` is pre-production step

Reply:
It's in the production build process, which is already pretty slow. It's an easy win for CI.

Reply:
@goldibex, I've created a separate github issue for this https://github.com/steemit/condenser/issues/1824